addons/website/data/website_data.xml

```diff
@@ -65,6 +65,102 @@
                 </t>
             </field>
         </record>
+
+        <record id="aboutus" model="ir.ui.view">
+            <field name="name">About Us</field>
+            <field name="type">qweb</field>
+            <field name="key">website.aboutus</field>
+            <field name="arch" type="xml">
+                <t name="About Us" t-name="website.aboutus">
+                    <t t-call="website.layout">
+                        <div id="wrap">
+                            <div class="oe_structure">
+                                <section class="s_title parallax s_parallax_is_fixed bg-black-50 pt24 pb24" data-vcss="001" data-snippet="s_title" data-scroll-background-ratio="1">
+                                    <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;"/>
+                                    <div class="o_we_bg_filter bg-black-50"/>
+                                    <div class="container">
+                                        <h1>About us</h1>
+                                    </div>
+                                </section>
+                            </div>
+                            <div class="oe_structure"/>
```

Comment:
purpose of the wizard is to have 'pre-build' page, why not content ? 
Anybody can create an empty page in 2 clicks and drop a parrallax :/

Reply:
Snippets are dynamically added to these pages in website.py > customize_page. The only purpose of adding this parallax is to have a custom title for the page (here: "About us").